kyuubi-server/src/test/scala/org/apache/kyuubi/engine/spark/SparkProcessBuilderSuite.scala

```diff
@@ -240,21 +240,22 @@ class SparkProcessBuilderSuite extends KerberizedTestHelper with MockitoSugar {
   }
 
   test("kill application") {
+    val engineRefId = "cf51ba6d-b019-4f8a-a2a2-4e479aa95112"
     val pb1 = new FakeSparkProcessBuilder(conf) {
       override protected def env: Map[String, String] = Map()
       override def getYarnClient: YarnClient = mock[YarnClient]
     }
-    val exit1 = pb1.killApplication("21/09/30 17:12:47 INFO yarn.Client: " +
-      "Application report for application_1593587619692_20149 (state: ACCEPTED)")
-    assert(exit1.contains("Killed Application application_1593587619692_20149 successfully."))
+    pb1.conf.set("spark.master", "yarn")
+    val exit1 = pb1.killApplication(engineRefId)
+    assert(exit1.equals(s"There are no Application tagged with $engineRefId"))
 
```

Comment:
If possible, add a successful kill application case here.

Reply:
Actually I don't know how to add yarn test cases

Reply:
FYI: MiniYarnService